src/ts/greedy-nav/GreedyNav.ts

```diff
@@ -184,6 +150,7 @@ export const updateLabel = (
   }
 
   toggle.innerHTML = label;
+  toggle.setAttribute('aria-label', `${label} navigation options`);
```

Comment:
I must admit, there are a couple of instances where `navDropdownBreakpointLabel` is used for the label, however, when I was manually testing it, the labels were only 'More' or 'Close'. I feel like we should completely refactor the greedy nav at some point in the future.

Reply:
➕ it is totally in need of an overhaul